examples/cosmos/lightclient/Blockchain.qnt

```diff
@@ -0,0 +1,168 @@
+//   This is a high-level specification of Tendermint blockchain
+//   that is designed specifically for the light client.
+//   Validators have the voting power of one. If you like to model various
+//   voting powers, introduce multiple copies of the same validator
+//   (do not forget to give them unique names though)
+
+module Blockchain {
+
+    pure def Min(a: int, b: int): int = if (a < b) a else b
+
+    import typedefs.* from "./typedefs"
+
+    // a set of all nodes that can act as validators (correct and faulty)
+    const AllNodes: Set[NODE]
```

Comment:
Shall we use `str` instead of `NODE`? I would prefer that for simplicity.

Reply:
sure, but this makes the spec a bit less accessible. Since we want engineers to read this spec, let's keep things simple.